routes/consts/informational_plate_state.js

```diff
@@ -0,0 +1,12 @@
+const PRESENT = 'Присутні';
```

Comment:
These consts already exist [root -> consts]
Why have you created them once more?